### PR TITLE
Migrate information about OpenShift SDN

### PR DESCRIPTION
**Pull request by jboxman**
- Adjust TOC tree
- Add content about OpenShift SDN modes

### CODE REVIEW

**Comment by jboxman** [comment]
@openshift/team-documentation, PTAL? Thanks!

**Comment by jboxman** [comment]
@vikram-redhat, pending peer review. SDN team provided feedback in a Google Doc.

**Comment by vikram-redhat** [comment]
Thanks @jboxman!

File: networking/openshift-sdn/about-openshift-sdn.adoc [hunk 1]
```diff
@@ -0,0 +1,27 @@
+[id="about-openshift-sdn"]
+= About OpenShift SDN
+include::modules/common-attributes.adoc[]
+:context: about-openshift-sdn
+
+toc::[]
+
+{product-title} uses a software-defined networking (SDN) approach to provide a
+unified cluster network that enables communication between pods across the
```
Comment (bergerhoffer) [inline networking/openshift-sdn/about-openshift-sdn.adoc]: Throughout the file, pod -> Pod

File: networking/openshift-sdn/about-openshift-sdn.adoc [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+{product-title} cluster. This pod network is established and maintained by the
+OpenShift SDN, which configures an overlay network using Open vSwitch (OVS).
+
+OpenShift SDN provides three SDN modes for configuring the pod network:
+
+* The _network policy_ mode allows project administrators to configure their own
+isolation policies using `NetworkPolicy` objects. Network policy is the default
+mode in {product-title} {product-version}.
+
+* The _multitenant mode_ provides project-level isolation for pods and services.
```
Comment (bergerhoffer) [inline networking/openshift-sdn/about-openshift-sdn.adoc]: In the other two bullets, you leave "mode" out of the italics - should be consistent.

File: networking/openshift-sdn/about-openshift-sdn.adoc [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+Pods from different projects cannot send packets to or receive packets from pods
+and services of a different project. You can disable isolation for a project,
```
Comment (bergerhoffer) [inline networking/openshift-sdn/about-openshift-sdn.adoc]: Throughout the file, servivce -> Service

File: networking/openshift-sdn/about-openshift-sdn.adoc [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+allowing it to send network traffic to all Pods and services in the entire
+cluster and receive network traffic from those Pods and services.
+
+* The _subnet_ mode provides a "flat" pod network where every pod can
+communicate with every other pod and service. The _network policy_ mode provides
```
Comment (bergerhoffer) [inline networking/openshift-sdn/about-openshift-sdn.adoc]: You've already introduced "network policy" before, so shouldn't italicize it here now.

File: networking/openshift-sdn/about-openshift-sdn.adoc [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+isolation policies using `NetworkPolicy` objects. Network policy is the default
+mode in {product-title} {product-version}.
+
+* The _multitenant mode_ provides project-level isolation for pods and services.
+Pods from different projects cannot send packets to or receive packets from pods
+and services of a different project. You can disable isolation for a project,
+allowing it to send network traffic to all Pods and services in the entire
+cluster and receive network traffic from those Pods and services.
+
+* The _subnet_ mode provides a "flat" pod network where every pod can
```
Comment (bergerhoffer) [inline networking/openshift-sdn/about-openshift-sdn.adoc]: Just wondering if the quotes around "flat" are necessary or not.

Comment (jboxman) [reply networking/openshift-sdn/about-openshift-sdn.adoc]: Oops, I don't think we do that anymore. This is most appropriated 3.x content.

**Comment by jboxman** [comment]
@bergerhoffer thanks for the review!

**Pull request merged**